### PR TITLE
Add README describing project

### PR DESCRIPTION
**Pull request by n-wach**

### CODE REVIEW

**Comment by n-wach** [comment]
I'm working on fixing the build checks below, probably in another branch.  Hopefully we can commit without passing those

**Comment by n-wach** [comment]
#5 needs to be merged first

File: README.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# ZoomTube
+
+View our [Design Doc](https://docs.google.com/document/d/12t7FDPDpTpHzrQBJ3T8qdmGKGrPi3lr2-UPmoc2ISHI).
```
Comment (amuamushu) [inline README.md:3]: I just enabled link sharing to anyone with a Google email. The shared link is slightly different, not too sure if the link in the readme needs to be updated.
```suggestion
View our [Design Doc](https://docs.google.com/document/d/12t7FDPDpTpHzrQBJ3T8qdmGKGrPi3lr2-UPmoc2ISHI/edit?usp=sharing).
```

File: README.md [hunk 2]
```diff
@@ -21,11 +21,12 @@ reply to other students’ questions.
...
 This project uses the following tools:
 
-- [YouTube Data API]() for generating transcripts
-- [YouTube iFrame API]() for displaying video
-- [AppEngine]() for deploying
-
-*TODO: Add urls and more APIs.*
+- [AppEngine on Google Cloud](https://cloud.google.com/appengine) for deploying
+- [Java Servlets](https://docs.oracle.com/javaee/5/tutorial/doc/bnafe.html) for back-end
+- [React](https://reactjs.org/) for front-end
```
Comment (amuamushu) [inline README.md:26]: Are we for sure using React? Also, should we include Bootstrap?

Comment (n-wach) [reply README.md:26]: If we decide on using React:
```suggestion
- [React](https://reactjs.org/) and [Bootstrap](https://react-bootstrap.github.io/) for front-end
```

Comment (n-wach) [reply README.md:26]: Remove React
```suggestion
```

**Comment by billie-cat** [comment]
Hi Nathan,
Fola and I should have been added to this PR.

**Pull request merged**